csharp/Samples/Microsoft.Spark.CSharp/App.config

```diff
@@ -11,31 +11,27 @@
     <root>
       <level value="DEBUG" />
       <appender-ref ref="ConsoleAppender" />
-      <!--
       <appender-ref ref="LogFileAppender" />
-      -->
     </root>
     <appender name="ConsoleAppender" type="log4net.Appender.ConsoleAppender">
       <layout type="log4net.Layout.PatternLayout">
         <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
       </layout>
     </appender>
-    <!--
     <appender name="LogFileAppender" type="log4net.Appender.RollingFileAppender">
       <file type="log4net.Util.PatternString">
-        <conversionPattern value="C:\\SparkCLRLogs\\SparkCLRSamplesLog_" />
+        <conversionPattern value="%env{TEMP}\\SparkCLRLogs\\SparkCLRSamples_%env{COMPUTERNAME}[%processid].log" />
       </file>
       <param name="AppendToFile" value="true" />
       <param name="MaxSizeRollBackups" value="2000" />
       <param name="MaxFileSize" value="51200000" />
       <param name="StaticLogFileName" value="false" />
-      <param name="DatePattern" value="yyyy_MM_dd_hh_mm" />
+      <param name="DatePattern" value="yyyy_MM_dd_hh" />
```

Comment:
The file names of generated logs look like `SparkCLRWorker_MININT-V8LJF0N[14592].log2015_11_30_04`, usually the log file name might look like `SparkCLRWorker_MININT-V8LJF0N[14592].log.2015_11_30_04`